package.json

```diff
@@ -148,6 +148,9 @@
     "testem-failure-only-reporter": "^0.0.1",
     "typescript": "^4.5.4"
   },
+  "resolutions": {
+    "typescript": "^4.5.4"
+  },
```

Comment:
Question about this: why not update `broccoli-typescript-compiler` instead?

Reply:
We're on the latest version. That said, it would make sense to send a PR to the library.

Reply:
Maybe https://github.com/tildeio/broccoli-typescript-compiler/pull/110 ?

Reply:
Sorry, yeah, I meant “make the change upstream” but that was totally unclear in my phrasing.